src/libraries/System.Reflection.MetadataLoadContext/tests/src/TestUtils/TestUtils.cs

```diff
@@ -253,7 +253,9 @@ public static void Dump(this CustomAttributeTypedArgument cta, int indent = 0)
 
         private static readonly Lazy<bool> s_useRuntimeTypesForTests = new Lazy<bool>(() =>
         {
-            if (File.Exists(Path.Combine(Path.GetDirectoryName(typeof(TestUtils).Assembly.Location), "UseRuntimeTypes.txt")))
+            var loc = AssemblyPathHelper.GetAssemblyLocation(typeof(TestUtils).Assembly);
+
+            if (File.Exists(Path.Combine(loc, "UseRuntimeTypes.txt")))
```

Comment:
Should we just return false for Browser? `loc` in Browser will be the assembly name + ".dll"